api/docs/installation/localvir.md

```diff
@@ -61,14 +62,19 @@ cp .env.example .env
 * To run the project on a local machine (default mode).
 
 ```sh
-(badgeyay) $ python app/main.py
+(badgeyay) $ export FLASK_APP=run.py
+
 ```
 
 * To run the project on a local machine (debug mode).
 
 ```sh
-(badgeyay) $ python app/main.py --dev
+(badgeyay) $ export FLASK_DEBUG=1
 ```
+* To run the project on local host.
+'''sh
```

Comment:
Can you combine this with the other two previous ones instead of putting this as a new one.